src/views/tickets/Screenshot.js

```diff
@@ -1,17 +1,36 @@
 import React, { Component } from 'react';
-import { Image, Segment } from 'semantic-ui-react';
+import { Image, Segment, Grid } from 'semantic-ui-react';
 import { getArtifactURL } from '../../utils/api.js';
 
 export default class Screenshot extends Component {
   render() {
-    return (
-      <Segment inverted>
+    let image;
+    //if there is a heigth/width apply it else the image is fluid
+    if (this.props.height != null && this.props.width != null) {
+      image = (
+        <Image
+          alt="fullpage screenshot"
+          src={getArtifactURL(this.props.ticketID, 'screenshotFull.png')}
```

Comment:
Because the filename is custom, the logic to pick out the filename will need to be a bit more complex

Reply:
That's definitely something to talk about today as well.

Reply:
Look at response for file artifacts, and loop and filter for .png (like js component)